com.chartboost.mediation.demo/Assets/Demo/Demo.cs

```diff
@@ -193,44 +199,31 @@ public async void OnShowFullscreenClick()
     #endregion
     
     #region Banners
-    private void SetupBannerDelegates()
-    {
-        ChartboostMediation.DidLoadBanner += DidLoadBanner;
-        ChartboostMediation.DidClickBanner += DidClickBanner;
-        ChartboostMediation.DidRecordImpressionBanner += DidRecordImpressionBanner;
-    }
 
-    public void OnCreateBannerClick()
+
+    public async void OnCreateBannerClick()
     {
+        if(_bannerAd != null)
+            Destroy(_bannerAd.gameObject);
+
+        var screenWidthNative = ChartboostMediationConverters.PixelsToNative(Screen.width);
+        
         var size = bannerSizeDropdown.value switch
         {
+            11 => ChartboostMediationBannerAdSize.Adaptive9X16(screenWidthNative),
+            10 => ChartboostMediationBannerAdSize.Adaptive1X4(screenWidthNative),
+            9 => ChartboostMediationBannerAdSize.Adaptive1X3(screenWidthNative),
+            8 => ChartboostMediationBannerAdSize.Adaptive1X2(screenWidthNative),
+            7 => ChartboostMediationBannerAdSize.Adaptive10X1(screenWidthNative),
+            6 => ChartboostMediationBannerAdSize.Adaptive8X1(screenWidthNative),
+            5 => ChartboostMediationBannerAdSize.Adaptive6X1(screenWidthNative),
+            4 => ChartboostMediationBannerAdSize.Adaptive4X1(screenWidthNative),
+            3 => ChartboostMediationBannerAdSize.Adaptive2X1(screenWidthNative),
             2 => ChartboostMediationBannerAdSize.Leaderboard,
             1 => ChartboostMediationBannerAdSize.MediumRect,
             _ => ChartboostMediationBannerAdSize.Standard
         };
-
-        _bannerAd?.Destroy();
-
-        Log("Creating banner on placement: " + bannerPlacementInputField.text + " with size: " + size);
-        _bannerAd = ChartboostMediation.GetBannerAd(bannerPlacementInputField.text, size);
         
-        if (_bannerAd == null)
-        {
-            Log("Banner not found");
-            return;
-        }
-
-        // example keywords usage
-        _bannerAd.SetKeyword("bnr_keyword1", "bnr_value1"); // accepted set
-        _bannerAd.SetKeyword("bnr_keyword2", "bnr_value2"); // accepted set
-        _bannerAd.SetKeyword(GenerateRandomString(65), "bnr_value2"); // rejected set
-        _bannerAd.SetKeyword("bnr_keyword3", GenerateRandomString(257)); // rejected set
-        _bannerAd.SetKeyword("bnr_keyword4", "bnr_value4"); // accepted set
-        var keyword4 = this._bannerAd.RemoveKeyword("bnr_keyword4"); // removal of existing
-        _bannerAd.RemoveKeyword("bnr_keyword4"); // removal of non-existing
-        _bannerAd.SetKeyword("bnr_keyword5", keyword4); // accepted set using prior value
-        _bannerAd.SetKeyword("bnr_keyword6", "bnr_value6"); // accepted set
-        _bannerAd.SetKeyword("bnr_keyword6", "bnr_value6_replaced"); // accepted replace
```

Comment:
Do we still want to keep the usage of keywords sample?

Reply:
Earlier, keywords were only stored on native so unity had to make an API call to set (or remove/replace) each key-value pair separately. With the new API keywords are stored in a C# dictionary so pubs can directly use the keywords as a dictionary. SInce it is a dictionary we don't have to provide example on how to use it since it will be just like using any other dictionary.

However, any update to this keywords will, however, require to first read the keywords from native, update it and then write it back to native. 

```
var keywords = _bannerAd.Keywords ??= new Dictionary<string, string>();   // read from native 
keywords.Add("bnr_keyword1", "bnr_value1"); 
keywords.Add("bnr_keyword2", "bnr_value2");
_bannerAd.Keywords = keywords;     // write to native
```

Reply:
If it is set to a new Dictionary, it will clear out the dictionary on C# but will not modify anything on native until you call `_bannerAd.Keywords = keywords` (where `keywords = new Dictionary`) then everything on native will also be deleted. 
I considered setting up an Observable Dictionary which would read and write on native as and when there is any update on C# but native implementation( checked on IOS) does it the same way ( read first, update and then write) so I didn't change anything on Unity